hw/dashboard/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Hardware Development Dashboard"
+---
+
+This page serves as the landing spot for all hardware development within the OpenTitan project.
+
+We start off by providing links to the [summary results of various tool-flows](#toolflows-summary-results) run on all of our [Comportable]({{< relref "doc/rm/comportability_specification" >}}) designs.
+This includes DV simulations, FPV and lint, all of which are run with the `dvsim` tool which serves as the common frontend.
+
+The [Comportable designs dashboard](#comportable-designs-dashboard) following it provides links to their design specifications, DV plans and tracks their current stage of development.
+See the [Hardware Development Stages]({{< relref "/doc/project/hw_stages.md" >}}) for description of the hardware stages and how they are determined.
+
+Next, we focus on all available [processor cores](#processor-cores) and provide links to their design specifications, DV plans and the DV simulation results.
+
+Finally, we provide the same set of information for all available [top level designs](#top-level-designs), while also including the synthesis results.
```

Comment:
I think we need to be a little careful about synthesis results, since we haven't
openly declared what we're synthesizing against. So either we need to hold
off on this or have explanatory text about what we're doing in the synthesis
world included here.

Reply:
I'd be OK with "eventually synthesis results"